device/cuda/CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ traccc_add_library( traccc_cuda cuda TYPE SHARED
   "src/fitting/fitting_algorithm.cu")
   
 target_compile_options( traccc_cuda
-  PRIVATE $<$<COMPILE_LANGUAGE:CUDA>:--expt-relaxed-constexpr> )
+  PRIVATE $<$<COMPILE_LANGUAGE:CUDA>:--expt-relaxed-constexpr -Xcompiler -finstrument-functions -fPIC -lnvToolsExt> )
```

Comment:
If possible, we should not add the `-lnvToolsExt` flag directly. Instead, it should be added using `target_link_libraries`.

Reply:
Done